### ISSUE TITLE
[Bug][A11Y][Badge] Warning icon/count should be black on yellow badge

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#470]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

Samsung

### OS Version

Android

### App version

0.6.0

### Describe the issue

Warning icon/count is white on yellow warning badge (light mode), contrast is not strong enough

<img width="193" height="80" alt="" src="https://github.com/user-attachments/assets/2279a6af-2327-4508-9a54-7f212b84716b" />

### Expected Behavior

black icon/count, as described in Figma 

<img width="303" height="64" alt="" src="https://github.com/user-attachments/assets/4bfb9b70-7c0c-4c81-8968-f02207eafa9f" />

**Comment by pya35** [comment]
seems to be fixed on 1.0.0, now we have black icon on yellow badge, please could you confirm @nouha06 ?

**Comment by nouha06** [comment]
> seems to be fixed on 1.0.0, now we have black icon on yellow badge, please could you confirm [@nouha06](https://github.com/nouha06) ?

Yes, it has been fixed since the badge component was updated to v1.2. The issue was caused by a missing token for the warning status, and it has already been resolved.
